README.md

```diff
@@ -29,5 +31,7 @@ OUTPUT=../output/report
   --region=${GOOGLE_CLOUD_REGION} \
   --tempLocation=${TEMP_LOCATION} \
   --inputFile=${INPUT_FILE} \
-  --output=${OUTPUT}"
+  --output=${OUTPUT} \
+  --allowMalformedRecords=${ALLOW_MALFORMED_RECORDS} \
+  --malformedRecordsMessage=${MALFORMEDRECORDSMESSAGE}"
```

Comment:
Under underscores to MLAFORMEDRECORDSMESSAGE (or maybe better MALFORMED_RECORDS_REPORT_PATH)